README.md

```diff
@@ -1,2 +1,89 @@
-# final-capstone-backend
-Final capstone backend
+# Easy booking
+
+> Project Description
+
+> The `EASY BOOKING` is a hotel booking app where users can make room reservations by specifying the date of entry and date for depature. Users will have various options to select from, based on what accomodities or special facilities offered along with a preferred room.
+
+## Built With
+- Ruby
+- Git
+- Github
+- Rubocop
+- RSpec
+- Stylelint
+
+
+## Getting Started
+To get a local copy of the program running, follow these simple steps
+## Demo
+Demo [link](https://easy-booking12.netlify.app/login)
+
+### Prerequisites
+- IDE, preferably Visual Studio Code
+
+### Install
+- Ruby
+- Rails
+
+
+### Setup
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone https://github.com/sntakirutimana72/final-capstone-backend to have a copy of the backend server.
+- To get the curent working tree, git fetch --all, then checkout to "dev" branch.
+- Finally, run the application by doing the following:
+  - navigate to the root directory with a command line preferably ubuntu terminal
+  - run 'bundle install'
+  - run 'rails db:create'
+  - run 'rails db:migrate'
+  - run 'rails db:seed' 
+  - run 'rails serve'
```

Comment:
- [ ] When I try to create the database following your instructions, I get this:

![image](https://user-images.githubusercontent.com/57421823/200048825-19a9703a-8e7e-4d32-80ed-d90d21e71fec.png)

It seems to be a problem with `:secret_key_base`. 

This means that I wasn't able to run the server, and the frontend without a running API to consume.

 If this is a bug, kindly fix it, if it's not, kindly include the solution in the instructions about how to run your app.

That way, your users will be able to run your app by following your instructions. 💪

Reply:
Dear @Meltrust 

Thank you for your feedback. It really meant a lot.
We've identified the issue as it got introduced when we were trying to deploy, and now we're working to fix it.